.github/workflows/update_build-tools-image.yml

```diff
@@ -0,0 +1,131 @@
+name: 'Update build tools image tag'
+
+# This workflow it used to update tag of build tools in ECR.
+# The most common use case is adding/moving `pinned` tag to `${GITHUB_RUN_IT}` image.
+
+on:
+  workflow_dispatch:
+    inputs:
+      from-tag:
+        description: 'Source tag'
+        required: true
+        type: string
+      to-tag:
+        description: 'Destination tag'
+        required: true
+        type: string
+        default: 'pinned'
+  pull_request:
```

Comment:
We don't want to run this on pull requests automatically
```suggestion
```

Reply:
already pushed the changes